languagetool-gui-commons/src/main/java/org/languagetool/gui/ConfigurationDialog.java

```diff
@@ -202,7 +233,35 @@ public void actionPerformed(@SuppressWarnings("unused") ActionEvent actionEvent)
     cons.fill = GridBagConstraints.HORIZONTAL;
     Collections.sort(rules, new CategoryComparator());
     rootNode = createTree(rules, true);  //  Style options
-    configTree[1] = new JTree(getTreeModel(rootNode));
+    configTree[1] = new JTree(getTreeModel(rootNode)){
```

Comment:
This code is duplicated, could you avoid that (maybe by adding a new class with the common code)?

Reply:
sorry,i will remove that.

Reply:
@danielnaber  Yes i tried it creating a separate class for this,but it threw an error.Will check once again and will confirm this.